datafusion/core/src/datasource/mod.rs

```diff
@@ -347,12 +347,5 @@ mod tests {
 
             Ok(RecordBatch::try_new(schema, new_columns).unwrap())
         }
-
-        fn map_partial_batch(
-            &self,
-            batch: RecordBatch,
-        ) -> datafusion_common::Result<RecordBatch> {
-            self.map_batch(batch)
-        }
```

Comment:
I don't know that there's a better way to deprecate this method. We could also just leave it?